Backend/router/v1/user.js

```diff
@@ -13,15 +13,34 @@ router.all('/', (req, res) => {
 });
 
 router.post('/signup', (req, res) => {
+
   let hashPassword = crypto.createHash("sha512").update(req.body.password).digest("hex");
 
-  models.user.create({
-    email: req.body.email,
-    password: hashPassword,
+  models.user.findAll({
+    where: {
+      email: req.body.email
+    }
+
   }).then(result => {
-    console.log("everything is good");
-    res.send(result);
+    if (result==""){
```

Comment:
usually it's easier to read if you add spaces around ==. "if (result == "")"

Reply:
@shberry I modified it.